docs/evaluation/robustness.md

```diff
@@ -0,0 +1,107 @@
+# Robustness Evaluation
+`robust_eval` is built on top of `ns eval` to evaluate the model on multiple benchmarks using different prompt variations.</br>
+ The purpose is to measure the **model robustness against changes in prompt** and then aggregate the results to metric stability and prediction consistency.
```

Comment:
add that the goal is to measure sensitivity to semantically equivalent prompt variant, no adversarial robustness. what's prediction consistency here?